src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -239,6 +247,51 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _defaultAppearanceViewModel;
     }
 
+    bool ProfileViewModel::HasUnfocusedAppearance()
+    {
+        return _profile.HasUnfocusedAppearance();
+    }
+
+    bool ProfileViewModel::EditableUnfocusedAppearance()
+    {
+        if constexpr (Feature_EditableUnfocusedAppearance::IsEnabled())
```

Comment:
This is the same as "return Feature_x::IsEnabled()"

Reply:
Curious: isn't this _technically_ faster because of the `if constexpr`? I suppose straight up returning `IsEnabled()` wouldn't be much slower haha, unless it's inline/constexpr? idk much about this stuff hahaha

Reply:
No. The compiler is smart enough to optimize...

```
if (true) { return true; }
return false;
```

into

```
return false;
```

on its own.